content/applications/sales/point_of_sale/shop/customer_display.rst

```diff
@@ -2,72 +2,102 @@
+#. Open the dropdown menu under the :guilabel:`Customer Display` section and select one of the
+   following options:
+
+   - :guilabel:`None`: To disable the customer display.
+   - :guilabel:`The same device`: To use the secondary screen connected with an HDMI or USB-C cable.
+   - :guilabel:`Another device`: To connect to a remote device connected to the database.
+   - :guilabel:`IOT-connected screen` for displays connected via an :doc:`IoT system
+     <../../../general/iot>`.
 
-To activate the feature, go to the POS settings, scroll down to the :guilabel:`Connected Devices`
-section, and tick the :guilabel:`Customer Display` checkbox.
+For displays connected using an :doc:`IoT system <../../../general/iot>`:
 
-.. image:: customer_display/feature-setting.png
-   :alt: customer display setting checkbox
+#. From the same section, tick the :guilabel:`IoT Box` checkbox to activate the IoT system in POS.
+#. Select the IoT-connected screen from the dropdown menu in the :guilabel:`Customer Display` field.
+
+Reaching the customer display
+=============================
 
 .. _customer_display/local:
 
 Local
 -----
 
-Connect a second screen to your POS using an HDMI or USB-C cable, then,
+To open the customer display on a second screen connected to a POS system using an HDMI or USB-C
+cable, follow these steps:
 
 #. :ref:`Open a POS session <pos/session-start>`.
 #. Click the :icon:`fa-bars` icon (:guilabel:`hamburger menu`).
-#. Click the :icon:`fa-desktop` icon (:guilabel:`customer screen`) to open a new window to drag and
-   drop onto the second screen.
+#. Click the :icon:`fa-desktop` icon (:guilabel:`customer screen`), which opens a new window to drag
+   onto the second screen.
 
 For POS terminals running the Odoo Android app with dual-screen support,
 
 #. :doc:`Activate the Point of Sale Mobile module <../../../general/apps_modules>` to enable the
    customer display.
+#. :ref:`Open a POS session <pos/session-start>`.
 #. Click the :icon:`fa-bars` icon (:guilabel:`hamburger menu`).
-#. Once installed, click the :icon:`fa-desktop` icon (:guilabel:`customer screen`) to open the
-   customer display on the terminal's secondary screen.
+#. Click the :icon:`fa-desktop` icon (:guilabel:`customer screen`), which opens the customer display
+   on the terminal's secondary screen.
 
 .. _customer_display/remote:
 
 Remote
 ------
 
-Access your database from another device (any computer, tablet, or smartphone), go to the POS
-application, click the vertical ellipsis button (:guilabel:`⋮`) on a POS card, and then
-:guilabel:`Customer Display` to open the display remotely.
+To open the customer display on a remote device, (any computer, tablet, or smartphone), follow these
+steps:
+
+#. Access your database from the other device.
+#. Navigate to :menuselection:`Point of Sale --> Dashboard`.
+#. Click the :icon:`fa-ellipsis-v` icon (:guilabel:`vertical ellipsis`) on a POS card.
+#. Click :guilabel:`Customer Display` to open the display remotely.
```

Comment:
```suggestion
#. Click the :icon:`fa-ellipsis-v` (:guilabel:`Dropdown menu`) icon on a POS card.
#. Click :guilabel:`Customer Display` to open the display remotely.
```

Reply:
I usually use the "tooltip" to describe an icon (even if it doesn't describe the icon itself). If there is no tooltip, I describe what it looks like (e.g., vertical ellipsis)